data/presets/amenity/townhall.json

```diff
@@ -21,6 +21,7 @@
     ],
     "terms": [
         "village",
+        "barangay",
```

Comment:
It's not necessary to repeat a "name" as  a term for the generic preset, if a dedicated preset exist with that name already. I'd remove it from here.

Reply:
Got that. Thank you.